src/main/java/edu/ucsb/cs156/happiercows/helpers/CommonStatsCSVHelper.java

```diff
@@ -0,0 +1,68 @@
+package edu.ucsb.cs156.happiercows.helpers;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+import edu.ucsb.cs156.happiercows.entities.CommonStats;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+/*
+ * This code is based on 
+ * <a href="https://bezkoder.com/spring-boot-download-csv-file/">https://bezkoder.com/spring-boot-download-csv-file/</a>
+ * and provides a way to serve up a CSV file containing information associated
+ * with an instructor report.
+ */
+
+public class CommonStatsCSVHelper {
+
+  private CommonStatsCSVHelper() {}
+
+  /**
+   * This method is a hack to avoid a jacoco issue; it isn't possible to 
+   * exclude an individual method call from jacoco coverage, but we can
+   * exclude the entire method.  
+   * @param out
+   */
```

Comment:
Sorry about this, but I goofed when I wrote the comment you copied this from; I should have said pitest.  Let's change this to the following so that it will be easier for future devs to see what's going on.
```suggestion
  /**
   * This method is a hack to avoid a pitest issue; it isn't possible to 
   * exclude an individual method call from pitest coverage, but we can
   * exclude the entire method by name in the pitest settings in pom.xml
   * @param out stream to close
   * @param csvPrinter printer to close
   */
```